recipes/reaktplot/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - 10.14                  # [osx]
+c_compiler:                # [win]
+  - vs2019                 # [win]
```

Comment:
`vs2019` seems to be the default now, so maybe specifying it here isn't necessary: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L10